rpc-jobs/RPC-Artifact-Build.yml

```diff
@@ -92,7 +96,15 @@
           auth-id: "github_account_rpc_jenkins_svc"
           status-context: 'CIT/artifact-{image}-{context}'
     dsl: |
-      // CIT Slave node
+      node('ArtifactBuilder2') {{
+        dir("rpc-gating") {{
+            git branch: env.RPC_GATING_BRANCH, url: env.RPC_GATING_REPO
+            common = load 'pipeline-steps/common.groovy'
+            aio_prepare = load 'pipeline-steps/aio_prepare.groovy'
```

Comment:
I think it's better to keep all the imports in the job definition instead of spreading them all around in the groovy files. Well, I think we should stick to one standard as that makes reviewing and maintaining a lot easier IMO.